crates/sui-transactional-test-runner/src/args.rs

```diff
@@ -92,6 +92,16 @@ pub struct ProgrammableTransactionCommand {
     pub inputs: Vec<ParsedValue<SuiExtraValueArgs>>,
 }
 
+#[derive(Debug, clap::Parser)]
+pub struct UpgradePackageCommand {
+    #[clap(long = "package")]
+    pub package: Option<String>,
```

Comment:
I think we'll want / need some input for package ID to upgrade (on the command line we use path). I made it optional in case we can derive this from previous `publish` commands in txn tests but not sure if that's feasible. In which case, this may have to be a required arg?